pom.xml

```diff
@@ -1,45 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.eclipse.jetty.toolchain</groupId>
     <artifactId>jetty-toolchain</artifactId>
-    <version>1.7</version>
+    <version>1.8-parent-SNAPSHOT</version>
   </parent>
-  <modelVersion>4.0.0</modelVersion>
   <groupId>org.eclipse.jetty.toolchain.setuid</groupId>
   <artifactId>jetty-setuid-jna</artifactId>
   <version>2.0.3-SNAPSHOT</version>
-  <name>Jetty :: SetUID JNA</name>
   <packaging>jar</packaging>
+  <name>Jetty :: SetUID JNA</name>
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
```

Comment:
This syntax (introduced in Maven 3.6.1) has broken several IDE integrations.
(mainly because the official maven xsd wasn't versioned and the IDEs were surprised by the updated xsd to support this)

IIRC, users on older maven versions will see odd failures if this is in use.

Why can't we just use the more normal ...

``` xml
<properties>
    <project.directory>${project.artifactId}</project.directory>
</properties>

<scm>
    <developerConnection>scm:git:git@github.com:jetty/</developerConnection>
</scm>
```

oh, as I write this I realize the artifactId and the repo name are not in sync. *grumble*

Reply:
I will remove those attributes as my first idea was simply to not have any `<scm>` and just setup this `<jetty.git.repo>jetty.setuid</jetty.git.repo>`
But I figure this will probably not work with m-release-p

Yes I would definitely be happy to use `${project.artifactId}` but the git repos have not been created with such naming conventions...